src/components/JobPost/Tags.tsx

```diff
@@ -0,0 +1,33 @@
+import React , {useState} from 'react'
+
+const Tags = () => {
+    // const [tags, setTags] = useState([]);
+    // const addTags = event => {
+    //     if (event.key === "Enter" && event.target.value !== "") {
+    //         setTags([...tags, event.target.value]);
+    //         event.target.value = "";
+    //     }
+    // };
+    return (
+//         <div className="tags-input">
+//             <ul>
+//                 {tags.map((tag, index) => (
+//                     <li key={index}>
+//                         <span>{tag}</span>
+//                         <i className="material-icons">close</i>
+//                     </li>
+//                 ))}
+//             </ul>
+//             <input
+//     type="text"
+//     onKeyUp={event => addTags(event)}
+//     placeholder="Press enter to add tags"
+// />
+//         </div>
+<>
+hello
+</>
+
+    );
+}
+export default Tags;
```

Comment:
Do we need this component ?

Reply:
I am working on tag component , getting typescript error

Reply:
Then you can remove the component for now to merge the current working code